### ISSUE TITLE
config parsing isn't working on macos.

### ISSUE THREAD

**Issue by jjr840430** [riboseinc/retrace#267]
The following configuration lines are working well on Ubuntu, but not on mac os.
Here is the example for configuration.

```
$ can examples/stringinject/stringinject.conf
logging-global,LOG_GROUP_NET,LOG_LEVEL_ALL
stringinject,INJECT_FORMAT_STR,send,3:RANDOM,1

```
On mac os, retrace prints the logs for whole syscall functions and doesn't parse stringinject configuration.

**Comment by ronaldtse** [comment]
@jjr840430 is this fixed by #277 already?

**Comment by jjr840430** [comment]
yes, @ronaldtse, It was fixed by that.
